### PR TITLE
Add RNASEQ specific config for MPCDF

### PR DESCRIPTION
**Pull request by jfy133**
As with discussion with @maltesemike and @ilight1542

### CODE REVIEW

File: conf/pipeline/eager/mpcdf.config [hunk 1]
```diff
@@ -1,64 +1,121 @@
...
+      params {
+        // Specific nf-core/configs params
+        config_profile_contact = 'James Fellows Yates (@jfy133)'
+        config_profile_description = 'nf-core/eager MPCDF raven profile provided by nf-core/configs'
+      }
+    process {
+
+      withName: malt {
+        maxRetries = 1
+        memory = 20000000.MB
```
Comment (apeltzer) [inline conf/pipeline/eager/mpcdf.config:73]: 20TB memory, really?

Comment (jfy133) [reply conf/pipeline/eager/mpcdf.config:73]: I wish...

File: docs/mpcdf.md [hunk 2]
```diff
@@ -12,24 +12,38 @@ All profiles use `singularity` as the corresponding containerEngine. To prevent
...
 
-To use: `-profile raven,mpcdf`
+To use: `-profile mpcdf,raven`
 
 Sets the following parameters:
 
 - Maximum parallel running jobs: 8
-- Max. memory: 368.GB
-- Max. CPUs: 192
+- Max. memory: 200000.MB (2.TB)
```
Comment (apeltzer) [inline docs/mpcdf.md:35]: This is 0.2TB ?

Comment (apeltzer) [reply docs/mpcdf.md:35]: Why not writing 2000.GB? to make it easier to read?

File: conf/pipeline/eager/mpcdf.config (hunk 1 above)
Comment (jfy133) [inline conf/pipeline/eager/mpcdf.config:73]:
```suggestion
        memory = 2000000.MB
```

Comment (apeltzer) [reply conf/pipeline/eager/mpcdf.config:73]: Also maybe if this is really true consder using at least the `.GB` unit, to make it easier to read ;-)

File: conf/mpcdf.config [hunk 3]
```diff
@@ -61,7 +61,7 @@ profiles {
        
         params {
             config_profile_description = 'MPCDF raven profile (unofficially) provided by nf-core/configs.'
-            max_memory = 368.GB
+            max_memory = 2000000.MB
```
Comment (apeltzer) [inline conf/mpcdf.config:64]: Maybe use GB as units and make it easier to read that way ;-)

File: docs/mpcdf.md (hunk 2 above)
Comment (jfy133) [inline docs/mpcdf.md:35]:
```suggestion
- Max. memory: 2000000.MB (2.TB)
```

Comment (jfy133) [reply docs/mpcdf.md:35]: For clarity - we specify with MB as apparently there was some wierd overhead that's not included and 2TB doesn't include that overhead (or something), but 2000000.MB is low enough away from a full 2TB to work

:shrug: it's what the MPCDF admins recommended to @maltesemike

File: conf/mpcdf.config (hunk 3 above)
Comment (maltesemike) [reply conf/mpcdf.config]: Unfortunately nextflow seems to interpret/calculate GB differently, and ends up exceeding the strict MB limit allowed on this cluster...

File: conf/pipeline/eager/mpcdf.config (hunk 1 above)
Comment (jfy133) [reply conf/pipeline/eager/mpcdf.config]: Doing it for clarity as that' exactly wat we specify

File: conf/mpcdf.config (hunk 3 above)
Comment (apeltzer) [inline conf/mpcdf.config]:
```suggestion
            max_memory = 2.TB
```

File: conf/pipeline/eager/mpcdf.config (hunk 1 above)
Comment (apeltzer) [inline conf/pipeline/eager/mpcdf.config]:
```suggestion
        memory = 2.TB
```

File: docs/mpcdf.md (hunk 2 above)
Comment (apeltzer) [reply docs/mpcdf.md]: 🤦🏻 Fair enough then,

File: conf/mpcdf.config (hunk 3 above)
Comment (apeltzer) [reply conf/mpcdf.config]: Future reference: Likely the Terabyte vs Tebibyte discussion https://www.dataunitconverter.com/terabyte-to-tebibyte/1

**Pull request merged**